packages/papaya/backend/sirius-components-papaya/src/main/java/org/eclipse/sirius/components/papaya/impl/PapayaFactoryImpl.java

```diff
@@ -554,7 +554,6 @@ public PapayaPackage getPapayaPackage() {
     /**
      * <!-- begin-user-doc --> <!-- end-user-doc -->
      *
-     * @deprecated
      * @generated
```

Comment:
Is it normal to have @deprecated here. I don't know. Perhaps we don't have the same version of EMF generation